packages/tools/src/tools/annotation/ContourROITool.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  getEnabledElement,
+  Types,
+  utilities as csUtils,
+  StackViewport,
+  VolumeViewport,
+} from '@cornerstonejs/core';
+import registerContourDrawLoop from './contourROITool/drawLoop';
+import { EventTypes, PublicToolProps, ToolProps } from '../../types';
+import PlanarFreehandROITool from './PlanarFreehandROITool';
+import { getViewportIdsWithToolToRender } from '../../utilities/viewportFilters';
+import triggerAnnotationRenderForViewportIds from '../../utilities/triggerAnnotationRenderForViewportIds';
+import InterpolationManager from '../../utilities/contourROITool/InterpolationManager';
+import { InterpolationROIAnnotation } from '../../types/ToolSpecificAnnotationTypes';
+import * as annotation from '../../stateManagement/annotation';
+
+const { getImageSliceDataForVolumeViewport } = csUtils;
+
+class ContourROITool extends PlanarFreehandROITool {
```

Comment:
Could we call this InterpolatationROITool since you call the types interpolation?
Or maybe InterpolationFreehandROITool to mirror  `PlanarFreehandROITool`